src/Model/NoDriverManager.php

```diff
@@ -38,7 +38,7 @@ public function findAll(): array
      * @param int|null $limit
      * @param int|null $offset
      */
-    public function findBy(array $criteria, ?array $orderBy = null, $limit = null, $offset = null): array
```

Comment:
I don't think is necessary to remove the return type

Reply:
Re added